contracts/interfaces/IMarketCommon.sol

```diff
@@ -13,8 +13,9 @@ interface IMarketCommon {
   struct EpochInfo {
     uint32 latestExecutedEpochIndex;
     // Reference to Chainlink Index
-    // NOTE: a future version of the protocol could rather require keeper to pass this in each epoch,it is unknown if verifying it each time with the orocle manager would be cheaper.
     uint80 latestExecutedOracleRoundId;
+    // This should be large enough for all price data.
+    uint144 lastEpochPrice;
```

Comment:
It makes sense to store the previous price in the contract - this means that we don't need to query chainlink each time for the previous price.

It also means that `uint80 latestExecutedOracleRoundId` isn't needed at all - but keeping it there because it doesn't cost much and is kinda nice to have as something that we can just query on chain.